distribution/docs/src/main/resources/_contents/_catalog-contents/integrating-catalog-contents.adoc

```diff
@@ -1135,7 +1135,11 @@ The configurable properties for the FTP Endpoint are accessed from the *FTP Endp
 
 ====== Using the endpoint
 
-The FTP endpoint supports the PUT and MPUT operations. Any other operation, such as DELETE, will return a 550 action not taken response.
+The FTP endpoint supports the PUT, MPUT DELE, RETR, RMD, APPE, RNTO, STOU, and SITE operations.
+
+The FTP endpoint supports files being upload as a dot-file (eg. .foo) and then being renamed to
+the final filename (eg. some-file.pdf). The endpoint will complete the ingestation process when
+the rename command is sent.
 
```

Comment:
I've added this content to https://github.com/codice/ddf/pull/1224 so it'll be in the new outline